projectcharter.md

```diff
@@ -0,0 +1,84 @@
+# forms.gov Project Charter
+Constituents of the Federal government use some number of more than 23,000 different forms available to them to interact with agencies; these agencies use the information from these forms to provide information or benefits, but spend innumberable hours recreating and quality checking handwritten entries on the original paper forms. This method of manual form processing based on paper-only forms leads to more than 11.4 billion hours of paperwork annually. As we have seen in multiple USDS engagements, web-based forms created through user-centered design methods reduce the respondent burden while producing well-structured data to the agencies for ingestion. This project creates a foundation for optimizing the process of building more useful, modern, and frictionless web-based forms.
+  
+  
+## Stakeholders
+#### Who needs to be informed about progress in major decisions?
+* Sponsor - Suzette King (OFCIO)
+* USDS Leadership
+* DHS Digital Service team
+* Others?
```

Comment:
I might add the Vets.gov team; this is where the library was originally created, and as Vets.gov is the only team currently using this tool in production, so any changes or enhancements we make to it will have to continue to work for them.